### PR TITLE
User can create an entry

### PR DESCRIPTION
**Pull request by mosesr-kim**
image

### CODE REVIEW

**Comment by mosesr-kim** [comment]
Screen Shot 2021-05-25 at 11 25 31 AM

**Comment by mosesr-kim** [comment]
Screen Shot 2021-05-25 at 11 28 10 AM
Screen Shot 2021-05-25 at 11 28 17 AM

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.head {
+  font-size: 24px;
+  font-weight: 400;
+}
+
+.container-main {
+  width: 50%;
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: We already made a `.container` class for you guys to use in the `layout.css` file, you should just use that one so you don't have multiple tools for one situation

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,24 @@
 /* global data */
 /* exported data */
+var $photoURL = document.querySelector('.photoURL');
+var $image = document.querySelector('.image');
+$photoURL.addEventListener('input', function (event) {
+  $image.setAttribute('src', $photoURL.value);
+});
+
+var inputs = {};
```
Comment (codingcodymiller) [inline js/main.js]: You should move this inside the function definition for submit, it doesn't need to exist after that function ends

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start!! A couple notes for you:

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**